### ISSUE TITLE
Admonition blocks constructed as a table and the title "note" is in CSS

### ISSUE THREAD

**Issue by neal-timpe** [redhat-documentation/redhat-docs-template#7]
In VoiceOver, the Admonition blocks do not read "Note" or "Important," which are the admonition box titles. It says "newline" instead"

When the CSS is turned off, the "Note" disappears completely. 

The admonition box is constructed as a a table, which is. 

I think these two failures are applicable.

https://www.w3.org/WAI/WCAG21/Techniques/failures/F34

https://www.w3.org/WAI/WCAG21/Techniques/failures/F49

Felicia noted this in another issue, maybe for DXP. I will try to find it and link it here.
